tests/cnf/core/network/dpdk/tests/rootless.go

```diff
@@ -252,7 +251,13 @@ var _ = Describe("rootless", Ordered, Label(tsparams.LabelSuite), ContinueOnFail
 				clientPodNetConfig,
 				sleepCMD,
 			)
-			rxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapOneInterfaceName, "${PCIDEVICE_OPENSHIFT_IO_DPDKPOLICYONE}"))
+
+			Expect(clientPod.WaitUntilRunning(time.Minute)).ToNot(HaveOccurred(), "Fail to wait until pod is running")
+			err = cmd.RxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapOneInterfaceName,
+				"${PCIDEVICE_OPENSHIFT_IO_DPDKPOLICYONE}"))
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
```

Comment:
Remove  fmt.Sprintf,  the error will shown anyway

Reply:
removed